docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst

```diff
@@ -317,8 +317,7 @@ If you are not interested in this additional functionality, the commands describ
 Prepare Bareos database
 -----------------------
 
-We assume that you have already your database installed and basically running.
-Using the PostgreSQL database backend is recommended.
+We assume that you already have your database server installed and basically running.
```

Comment:
```
We assume that you already have your PostgreSQL database server installed and basically running.
```
So basically we remember any reader that only PostgreSQL is actually supported. (Perhaps he got previous informations about the support of different backend)

Reply:
What is your point here? Do you propose a change?

Reply:
you already have your **PostgreSQL** database server

Reply:
Okay, added.